Events/src/test/java/com/github/swcraftlyon/meetup/infra/repositories/PlannerRepositoryTest.java

```diff
@@ -0,0 +1,18 @@
+package com.github.swcraftlyon.meetup.infra.repositories;
+
+import com.github.swcraftlyon.meetup.domain.IPlanner;
+import com.github.swcraftlyon.meetup.domain.entities.Event;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+
+public class PlannerRepositoryTest {
+
+    @Test
+    public void shouldRecordAnEvent() throws Exception {
+        IPlanner planner = Mockito.mock(IPlanner.class);
+        Event event = new Event("Mix-it Lyon");
+        planner.record(event);
+        Mockito.verify(planner).record(event);
```

Comment:
In this case, I wanted to know what is the best way to test the record method . because this one doesn't do a lot of things instead of filling a document and invoke the persist of the DAO class. 

Ussally i write an integration test in this case. thank's for your help.